### PR TITLE
Clean up files

### PR DESCRIPTION
**Pull request by ronanfitz**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_SERVER_URL=http://localhost:8001
```
Comment (HamidAghdaee) [inline .env]: I believe this should go into .env.development and there should be a corresponding .env.production

File: .env [hunk 2]
```diff
@@ -1,2 +1 @@
-REACT_APP_API_SERVER_URL=http://localhost:8001
```
Comment (HamidAghdaee) [inline .env]: this the only change I see for this PR. Why this PR at all? Also if you remove this line, you need to put it inside .env.development

**Comment by HamidAghdaee** [comment]
@ronanfitz Is this PR out of date? is it no longer relavant?

**Comment by ronanfitz** [comment]
I believe so. Yes.

**Pull request closed without merging**